echemdb/data/cv/entry.py

```diff
@@ -207,12 +208,20 @@ def y(self):
         """
         from astropy import units as u
 
-        if u.Unit(self.figure_description.current.unit).is_equivalent('A / m2'):
-            return 'j' 
-        if u.Unit(self.figure_description.current.unit).is_equivalent('A'):
-            return 'I'
-        else:
-            raise ValueError(f"The variable on the y-axis is not equivalent to 'A / m2' or 'A'.")
+        if not self.data_description.axes:
+            raise IndexError("No axes were specified for the dataset.")
+        if not 'I' or not 'j' in self.data_description.axes._descriptor.keys():
+            raise KeyError("None of the axes has a variable 'I' or 'j'.")
+        if {'j', 'I'} <= self.data_description.axes._descriptor.keys():
+            raise KeyError("The axes should only contain the variable 'I' or 'j', but not both.")
```

Comment:
This won't work. The condition evaluates as `not 'I'` (which is False because `'I'` is True) `or` the other part.

```suggestion
        if len({'j', 'I'}.intersection(self.data_description.axes._descriptor.keys())) == 1:
            raise KeyError("Exactly one axis must be labeled 'I' or 'j'.")
```